src/Repository/RunningMateRequestRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\ApplicationRequest\RunningMateRequest;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\Query\Expr\Join;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class RunningMateRequestRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, RunningMateRequest::class);
+    }
+
+    public function findForReferent(Adherent $referent): array
```

Comment:
what about return just an array of the object (without `is_adherent` level)?
```php
$data = [];
foreach ($result as $row) {
  $row[0]->setAdherent($row['is_adherent']);
  $data[] = $row[0];
}
return $data;
```

Reply:
@ottaviano this column is needed to be displayed in the list.